test/functional/apps/discover/_shared_links.js

```diff
@@ -1,3 +1,22 @@
+/*
+* Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
oops, looks like we added this in the backport.  don't see any problem with it though, up to you if you want to remove it